website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx

```diff
@@ -18,32 +18,29 @@ Policy sets are collections of policies you can apply globally or to specific [p
 
 To view and manage policies and policy sets, you must have [manage policy permissions](/terraform/cloud-docs/users-teams-organizations/permissions#manage-policies) for your organization.
 
-## Differences between Policy Checks and Policy Evaluations
+## Comparing policy checks and evaluations
 
-Though policy checks and policy evaluations seem like synonyms, there is a fundamental difference between the policy check and policy evaluation workflow.
+Sentinel policy sets can run as both policy checks and evaluations, while OPA policy sets only run as evaluations. Policy checks and evaluations can access different types of data and enable slightly different workflows.
```

Comment:
> Sentinel policy sets can run as both policy checks and evaluations, while OPA policy sets only run as evaluations

Is this required since we already call this out in both the **Policy Checks** and **Policy Evaluations** sections below.